libs/langchain-cohere/src/embeddings.ts

```diff
@@ -70,8 +75,16 @@ export class CohereEmbeddings
       token: apiKey,
     });
     this.model = fieldsWithDefaults?.model ?? this.model;
+
+    if (!this.model) {
```

Comment:
Would prefer to just make this a required param and ship a new minor version release along with your tool calling stuff